src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/OffenderTemporaryAbsenceReturnRepository.kt

```diff
@@ -7,6 +9,7 @@ import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.OffenderTemporaryAbsenc
 
 @Repository
 interface OffenderTemporaryAbsenceReturnRepository : JpaRepository<OffenderTemporaryAbsenceReturn, OffenderExternalMovementId> {
-  @Suppress("ktlint:standard:function-naming")
```

Comment:
Do we just not need this any more despite the `_` in the name?

Reply:
Should have been moved to the top of the file